bootq/src/main/java/life/genny/bootq/utils/GoogleSheetBuilder.java

```diff
@@ -344,6 +348,13 @@ public QuestionQuestion buildQuestionQuestion(Map<String, String> row, String re
 		questionQuestion.setIcon(icon);
 		questionQuestion.setRealm(realmName);
 		questionQuestion.setParentId(parent.getId());
+
+        String capReqs = row.get("capreqs");
+        if(capReqs != null) {
```

Comment:
This is good, but shouldn't we be doing the same for EntityAttributes?

Reply:
This is an update to get the add items menu working (capreqs has an immediate need to be on question question, however the rest could be done in a separate PR

Reply:
Sounds good.